scico/loss.py

```diff
@@ -261,13 +161,15 @@ def __init__(
         else:
             self.is_smooth = None
 
-        if isinstance(self.A, linop.LinearOperator):
```

Comment:
This seems to have been a serious oversight in out discussion of removing the `is_quadratic` attribute: `A` need not be a linear operator, and if it isn't, our assumptions about the loss function are violated. Is there any way of avoiding restoring this attribute?

Reply:
> A need not be a linear operator, and if it isn't, our assumptions about the loss function are violated

Can you elaborate? It does not need to be a linop in terms of the admm?

Reply:
Ignore my comment. I had forgotten about the earlier `if not isinstance(admm.f.A, LinearOperator):` test.